rust/worker/src/tracing/opentelemetry_config.rs

```diff
@@ -37,9 +37,34 @@ pub(crate) fn init_otel_tracing(service_name: &String, otel_endpoint: &String) {
     let stdout_layer =
         BunyanFormattingLayer::new(service_name.clone().to_string(), std::io::stdout)
             .with_filter(tracing_subscriber::filter::LevelFilter::INFO);
-    // global filter layer. Don't filter anything at global layer for this crate. And disable
-    // for every other library.
-    let global_layer = EnvFilter::new("none,worker=trace");
+    // global filter layer. Don't filter anything at above trace at the global layer for chroma.
+    // And enable errors for every other library.
+    let global_layer = EnvFilter::new(
+        "error,".to_string()
+            + &vec![
+                "chroma",
```

Comment:
everytime we add a crate, will we have to add it here?

Reply:
If we want it to be something other than error, yes.  There's no real good way to select by crate and then not have to select by crate.